src/core/index.md

```diff
@@ -1,63 +1,61 @@
-make up the family of core open-source projects.
+DataJoint Core is a collection of open-source tools for building and operating scientific data pipelines.
 
 ## APIs
 
 <div class="grid cards" markdown>
 
--   :fontawesome-brands-python:{ .lg .middle } **DataJoint Python**
+-   :fontawesome-brands-python:{ .lg .middle } **DataJoint for Python**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A Python client for defining, operating, and querying data pipelines.
 
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](https://docs.datajoint.org/python/)
+    :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/python/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)
 
--   :fontawesome-brands-java:{ .lg .middle } **DataJoint MATLAB**
+-   :fontawesome-brands-java:{ .lg .middle } **DataJoint for MATLAB**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A MATLAB client for defining, operating, and querying data pipelines.
 
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](https://docs.datajoint.org/matlab/).
+    :octicons-arrow-right-24: [Documentation](https://docs.datajoint.org/matlab/) | [Source code](https://datajoint.com/datajoint/datajoint-matlab)
 
 -   :fontawesome-solid-flask:{ .lg .middle } **Pharus**
 
     ---
 
-    Expose [data pipelines](./glossary#data-pipeline) via a REST interface.
-
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](./pharus/).
+    A REST API server for interacting with DataJoint pipelines.
 
+    :octicons-arrow-right-24: [Documentation](https://datajoint.github.io/pharus/) | [Source code](https://github.com/datajoint/pharus/)
+ 
 </div>
 
-## Web GUIs
+## Web Applicaitons
 
 <div class="grid cards" markdown>
 
+
 -   :fontawesome-brands-chrome:{ .lg .middle } **LabBook**
 
     ---
 
-    Data entry and data model browsing for [data pipelines](./glossary#data-pipeline).
+	A browser-based graphical user interface for data entry and navigation. 
 
-    :octicons-arrow-right-24: New docs coming soon!  In the meantime, refer to our [legacy docs](./datajoint-labbook/).
+    :octicons-arrow-right-24: [Documentation](https://datajoint.github.io/datajoint-labbook/) | [Source code](https://github.com/datajoint/datajoint-labbook/)
 
 -   :fontawesome-brands-chrome:{ .lg .middle } **SciViz**
 
     ---
 
-    A visualization framework for making low-code web apps for 
-    [data pipelines](./glossary#data-pipeline).
+    A framework for making low-code web apps for data visualization.
 
-    :octicons-arrow-right-24: New docs coming soon!
+    :octicons-arrow-right-24: [Source code](https://github.com/datajoint/sci-viz)
 
 </div>
 
-## Container Images
+## Docker Images
```

Comment:
Docker is a particular technology that provides a solution for containerizing workloads. We may continue using Docker or change it in the future. The more appropriate term here is container.

```suggestion
## Container Images
```